File: java/src/main/java/com/example/demo/commands/ListGreetingCommand.java
package com.example.demo.commands;

import java.util.List;

import com.example.demo.entities.Greeting;
import com.example.demo.services.GreetingService;

public class ListGreetingCommand implements ICommand{
    
    private final GreetingService greetingService; 

    public ListGreetingCommand(GreetingService greetingService) {
        this.greetingService = greetingService;
    }

    @Override
    public void invoke(List<String> tokens) {
        List<Greeting> glist = greetingService.getAllGreetings();
        System.out.println(glist);
    }

}

File: python/src/commands/ListGreetingCommand.py
from typing import List
from src.commands import ICommand
from src.services import GreetingService


class ListGreetingCommand(ICommand):
    def __init__(self, greetingService: GreetingService) -> None:
        self._greetingService = greetingService

    def invoke(self, tokens: List[str]) -> None:
        gList = self._greetingService.getAllGreetings()
        print(gList)